view/rooms/app.js

```diff
@@ -258,6 +265,13 @@ function setupSocketIOClient(bots) {
           detail: events,
         }));
       });
+
+      if (events.context.type === 'RoomState') {
+        document.getElementById('activeToggle')
```

Comment:
can you reuse the existing `activeToggle` const reference instead of doing document.getElementId every time?